### DISCUSSION TITLE
Dynmap support ?

### DISCUSSION THREAD

**Discussion by ZeLFaDore** [GriefPrevention/GriefPrevention#833] [Q&A]
Quick question does griefprevention support dynmap like to tell people that others has claimed this area like this shown on the pic and if yes how to connect them up 

Image : https://ibb.co/pJzKFbJ

**Comment by TomLewis** [comment]
https://www.spigotmc.org/resources/jdynmapgriefprevention.818/

https://dev.bukkit.org/projects/dynmap-griefprevention

**Comment by RoboMWM** [comment]
FrozenBeard has listed one of many addons that do this - I think even the dynmap dev himself has an addon for GP (unless it's that bukkit one - I've just seen his GitHub repo).

**Comment by fredagsguf** [comment]
Both of those addons are either outdated or creates enormous lag.
last update 2016, and 2015...

**Comment by TomLewis** [comment]
> 
> 
> Both of those addons are either outdated or creates enormous lag.
> last update 2016, and 2015...

Then update them.

**Comment by fredagsguf** [comment]
> > Both of those addons are either outdated or creates enormous lag.
> > last update 2016, and 2015...
> 
> Then update them.

If i knew how, don't you think i would have done it already?
